VKontakte/VKontakte/CustomView/LikeControl.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LikeControl.swift
+//  VkClone
+//
+//  Created by Edvard on 06.08.2021.
+//
+
+import UIKit
+
+final class LikeControl: UIControl {
+
+    private let countLabel = UILabel()
+    private let myButton: UIButton = {
```

Comment:
```suggestion
    private let myButton: UIButton = {
```
my ? - дай норм название